Building Sandstorm in a Debian Bullseye VM fails due to old golang-go version

When following the [build-from-source instructions](https://docs.sandstorm.io/en/latest/install/#option-4-installing-from-source), building Sandstorm from source fails because boringssl needs a golang-go version of 1.19 or greater:

```
…
# command-line-arguments
util/embed_test_data.go:79:16: undefined: os.ReadFile
util/embed_test_data.go:129:16: undefined: os.ReadFile
note: module requires Go 1.19
make[3]: *** [CMakeFiles/crypto_test_data.dir/build.make:302: crypto_test_data.cc] Error 2
make[3]: *** Deleting file 'crypto_test_data.cc'
make[3]: Leaving directory '/home/vagrant/sandstorm/deps/boringssl/build'
make[2]: *** [CMakeFiles/Makefile2:356: CMakeFiles/crypto_test_data.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
…
```

The [current version of golang-go in Bullseye](https://packages.debian.org/bullseye/golang-go) is 1.15.

I propose that we update the documentation to explain how to install a newer version of Go in Debian Bullseye.

--- a/docs/install.md
+++ b/docs/install.md
@@ -144,7 +144,7 @@ Please install the following:
 * `curl`
 * `python3`
 * `zlib1g-dev`
-* `golang-go`
+* `golang-go` 1.19 or later
 * `cmake`
 * discount (markdown parser)
 * [Meteor](http://meteor.com)
@@ -153,10 +153,29 @@ On Debian or Ubuntu, you should be able to get all these with:
 
     sudo apt-get install build-essential libcap-dev xz-utils zip \
         unzip strace curl discount git python3 zlib1g-dev \
-        golang-go cmake strace flex bison locales
+        cmake flex bison locales
+    GO_VERSION=$(curl 'https://go.dev/VERSION?m=text')
+    curl -L "https://go.dev/dl/$GO_VERSION.linux-amd64.tar.gz" -o go.tar.gz \
+        && sudo tar -C /usr/local -xvf go.tar.gz \
+        && rm go.tar.gz
     curl https://install.meteor.com/ | sh
 
-On Fedora 34 you should be able to get them with:
+When you download and install Go on Debian or Ubuntu using the instructions above, you should
+add the Go `bin` directory to your path:
+
+    export PATH="/usr/local/go/bin:$PATH"
+
+To add the Go `bin` directory to your path permanently, add it to your `~/.profile` file:
+
+    << EOF cat >> ~/.profile
+
+    # Include Go in the PATH
+    if [ -d /usr/local/go/bin ]; then
+        PATH="/usr/local/go/bin:\$PATH"
+    fi
+    EOF
+
+On Fedora 34 you should be able to get the prerequisites with:
 
     sudo dnf install make libcap-devel libstdc++-devel libstdc++-static \
        glibc-headers glibc-static glibc-locale-source gcc-c++ xz zip \

--- a/shell/i18n/en.i18n.json
+++ b/shell/i18n/en.i18n.json
@@ -298,7 +298,7 @@
       "whoHasAccess": "See who has access"
     },
     "grainSharePopup": {
-      "oldSharingModelDescription": "This grain uses the old sharing model. You can share it by copy/pasting the URL from the location bar",
+      "oldSharingModelDescription": "This grain uses the old sharing model. You can share it by copy/pasting the URL from the location bar.",
       "upgradeDescription": "You can upgrade this grain to the new model, but anyone with whom you shared this grain previously will lose access; you will have to share with them again.",
       "upgradeWarning": "Upgrading cannot be undone.",
       "upgrade": "upgrade",

